### PR TITLE
[ODS-4275] Convert EdFi.Ods.Tests.TestExtension to .NET Core

### PR DESCRIPTION
**Pull request by BhanuprakashThunga**

### CODE REVIEW

**Comment by BhanuprakashThunga** [comment]
@jamessmckay  Since the build is failing i will be working on this PR. Please verify this once i move the ticket to QA

**Comment by BhanuprakashThunga** [comment]
@jamessmckay  you can verify this now..and let me know if anything needs to be changed

File: Application/EdFi.Ods.Tests.TestExtension/EdFi.Ods.Tests.TestExtension.csproj [hunk 1]
```diff
@@ -14,41 +14,62 @@
...
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+    </Content>
+    <Content Include="Artifacts\**\Schemas\*.xsd">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+    </Content>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Artifacts\Metadata\ApiModel.json" />
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Tests.TestExtension/EdFi.Ods.Tests.TestExtension.csproj:49]: lets revert this please, and remove all the other embedded objects and code gen artifacts. in this case we just created a hard coded apimodel.json for this assembly

Comment (BhanuprakashThunga) [reply Application/EdFi.Ods.Tests.TestExtension/EdFi.Ods.Tests.TestExtension.csproj:49]: @jamessmckay  removed the embedded objects and code gen artifact dependencies

**Pull request merged**